resources/views/auth/application/status_update_component.blade.php

```diff
@@ -1,19 +1,18 @@
-<div class="right">
-    @foreach (\App\Models\Application::STATUSES as $st)
-    @if($st != \App\Models\Application::STATUS_IN_PROGRESS || user()->isAdmin())
-
-    <p style="margin:5px">
-        <label>
-            <input type="radio" wire:click="set('{{$st}}')" name="{{$this->application->id}}_status" @checked($this->application->status == $st) />
-            <span>@include('auth.application.status', ['status' => $st])</span>
-        </label>
-    </p>
-    @endif
-
-    @endforeach
-    @if (session()->has('message'))
-        <script>
-            M.toast({html: "{{ session('message') }}"});
-        </script>
-    @endif
+<div style="display: flex;flex-direction: column;align-items: center;justify-content: center;">
+    <label>
+        <input type="checkbox"
+               wire:click="callIn('{{$workshop->workshop_id}}')" @checked($workshop->called_in) />
+        <span>Behívva</span>
+    </label>
+    <label>
+        <input type="checkbox"
+               wire:click="admit('{{$workshop->workshop_id}}')" @checked($workshop->admitted) />
+        <span>Felvéve</span>
```

Comment:
I think these should be radio buttons; it's strange that we can admit students without calling them in. But I don't know what to call the option before 'called in'; so we can leave it as it is if we have no better idea.

Reply:
We have 3 states: nothing, called in, admitted. It's hard to make it even with radio buttons. I agree that technically we can make someone admitted but not called in, which might be strange, but the system will eventually handle it as admitted. I think we can live with that, or make it nicer later.

Reply:
OK; let's do it then later. Maybe we could open an issue for it.